server/svix-server/src/v1/endpoints/event_type.rs

```diff
@@ -70,6 +71,30 @@ impl ModelIn for EventTypeUpdate {
     }
 }
 
+#[derive(Deserialize, ModelIn, Serialize, Validate)]
+#[serde(rename_all = "camelCase")]
+struct EventTypePatch {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[validate(custom = "validate_no_control_characters")]
+    description: Option<String>,
+
+    #[serde(default, rename = "archived", skip_serializing_if = "Option::is_none")]
+    deleted: Option<bool>,
+
+    #[serde(default, skip_serializing_if = "NullablePatchField::is_absent")]
+    schemas: NullablePatchField<serde_json::Value>,
+}
+
+impl ModelIn for EventTypePatch {
+    type ActiveModel = eventtype::ActiveModel;
+
+    fn update_model(self, model: &mut Self::ActiveModel) {
+        patch_field_non_nullable!(self, model, description);
```

Comment:
Random idea, but to make it less error-prone we could maybe have:
```rust
let { description, deleted, schemas } = self;
```
Or whatever the syntax for deconstruction is, and then we'll be forced to use all of the field members below, no?